demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1727,46 +1767,52 @@ def re_create_id_set(id_set_path: Optional[str] = DEFAULT_ID_SET_PATH, pack_to_c
                 integration_list.extend(arr)
+                update_excluded_items_dict(excluded_items_from_id_set, excluded_items, excluded_items_by_type)
 
         progress_bar.update(1)
 
         if 'Playbooks' in objects_to_create:
             print_color("\nStarting iteration over Playbooks", LOG_COLORS.GREEN)
-            for arr in pool.map(partial(process_general_items,
-                                        print_logs=print_logs,
-                                        expected_file_types=(FileType.PLAYBOOK,),
-                                        data_extraction_func=get_playbook_data,
-                                        marketplace=marketplace,
-                                        ),
-                                get_playbooks_paths(pack_to_create)):
+            for arr, excluded_items in pool.map(partial(process_general_items,
+                                                        print_logs=print_logs,
+                                                        expected_file_types=(FileType.PLAYBOOK,),
+                                                        data_extraction_func=get_playbook_data,
+                                                        marketplace=marketplace
+                                                        ),
+                                                get_playbooks_paths(pack_to_create)):
                 for _id, data in (arr[0].items() if arr and isinstance(arr, list) else {}):
                     if data.get('pack'):
-                        packs_dict[data.get('pack')].setdefault('ContentItems', {}).setdefault('playbooks', []).append(_id)
+                        packs_dict[data.get('pack')].setdefault('ContentItems', {}).setdefault('playbooks', []).append(
+                            _id)
                 playbooks_list.extend(arr)
+                update_excluded_items_dict(excluded_items_from_id_set, excluded_items, excluded_items_by_type)
 
         progress_bar.update(1)
 
         if 'Scripts' in objects_to_create:
             print_color("\nStarting iteration over Scripts", LOG_COLORS.GREEN)
-            for arr in pool.map(partial(process_script,
-                                        print_logs=print_logs,
-                                        marketplace=marketplace,
-                                        ),
-                                get_general_paths(SCRIPTS_DIR, pack_to_create)):
+            for arr, excluded_items in pool.map(partial(process_script,
+                                                        print_logs=print_logs,
+                                                        marketplace=marketplace,
+                                                        ),
+                                                get_general_paths(SCRIPTS_DIR, pack_to_create)):
                 for _id, data in (arr[0].items() if arr and isinstance(arr, list) else {}):
```

Comment:
please fix everywhere
```suggestion
                for _id, data in (arr[0] if arr and isinstance(arr, list) else {}).items():
```